apps/happy-blocks/block-library/support-content-footer/style.scss

```diff
@@ -11,8 +11,7 @@ $system-font-family: -apple-system, BlinkMacSystemFont, "Noto Sans", "Segoe UI",
 	max-width: none !important;
 	align-items: normal !important;
 	box-sizing: border-box;
-	padding-bottom: 20px;
-	padding-top: 20px;
+	padding: 20px 24px;
```

Comment:
Could you try without this change? I see the padding correctly even without changing this, I think it would be better so this can inherit the horizontal padding from other styles!

Reply:
This padding issue is only visible on the tablet viewport. It gets like this 
<img width="802" alt="image" src="https://github.com/Automattic/wp-calypso/assets/2653810/e4e5562f-ea35-450a-a6b0-0c7ecbfc3ef0">
Could you please verify if you see it like this before this PR changes?

Reply:
![image](https://github.com/Automattic/wp-calypso/assets/52076348/03af6452-9bed-42f0-aff0-4e30f5431a19)

I see this in production, padding coming from [here](https://github.com/Automattic/wpsupport3/blob/feat/add_breadcrumbs_learn/styles/templates/_style.scss#L56). What are you using as width/device for testing?

Reply:
Sorry for the lack of information, support is fine, this issue is visible on learn since they have different HTML structure.

On support, the footer content is inside `<main>` which sets the padding. Learn, on the other hand, has it outside, this is why this issue is surfacing on `/learn`

Reply:
Oooh ok then!

Reply:
Also, posts pages on /support have it outside `<main>`. These cases will also be fixed. 👍